src/FixedSizeArrays.jl

```diff
@@ -2,7 +2,10 @@ VERSION >= v"0.4.0-dev+6521" && __precompile__(true)
 module FixedSizeArrays
 
 importall Base
-import Base.Func, Base.LinAlg.chol!
+import Base.LinAlg.chol!
+
+# for 0.5 and 0.4 compat, use our own functor type
+abstract Functor{N}
```

Comment:
I was thinking about doing this as part of the promotion changes to ops.jl, but now I'm wondering if it'll interfere with our ability to make `a + b` return the right type when `a::Vector{Point{N,T}}` and `b::Vector{Vec{N,T}}`?